lib/collection/src/collection/shard_transfer.rs

```diff
@@ -357,4 +356,20 @@ impl Collection {
             }
         }
     }
+
+    /// Whether we have reached the shard transfer soft limit based on the given incoming and
+    /// outgoing transfers.
+    pub(super) fn check_shard_transfer_limit(&self, incoming: usize, outgoing: usize) -> bool {
```

Comment:
Probably same comment here
```suggestion
    pub(super) fn check_automatic_shard_transfer_limit(&self, incoming: usize, outgoing: usize) -> bool {
```